packages/nimble-components/src/theme-provider/design-tokens.ts

```diff
@@ -170,6 +171,10 @@ export const borderHoverColor = DesignToken.create<string>(
     styleNameFromTokenName(tokenNames.borderHoverColor)
 ).withDefault((element: HTMLElement) => getColorForTheme(element, DigitalGreenLight, DigitalGreenLight, White));
 
+export const ozoneLibraryBackgroundColor = DesignToken.create<string>(
```

Comment:
Hi @WarriorOfNI can you give some more context for adding this token?

Some thoughts:
- The Nimble design system is for building a consistent style across NI applications. Right now we have the applicationBackgroundColor token to represent the background color of applications in different themes.
- We should not add application specifc tokens to nimble. Nimble should be used to capture consistency across applications. What does this token represent that should be shared across multiple applications?

Some questions:
- Is this token intended to be used for the application background color? Is so, why are we not using the applicationBackbroundColor to be consistent across application?
- Can you include screenshots and a description of where the token is used in an application?

If it helps you can reach out on teams to discuss your use cases further. Unfortunately your GitHub profile does not include a name for me to know who to reach out to.

Reply:
Paul-Andre reached out to me on Teams which helped me connect some dots. This is coming from [a comment I left on this Skyline repo design document](https://dev.azure.com/ni/DevCentral/_git/Skyline/pullrequest/431090?_a=files&path=%2FSpecs%2FDokker%2FEssential%20UI%20Styling%28Login%20Page%20and%20Mstr%20Library%29%2FEssential%20UI%20Styling%28Login%20Page%20and%20Mstr%20Library%29.md&discussionId=4122010) where I requested the Ozone app use more Nimble tokens and less hardcoded colors. 

I hadn't fully understood the purpose of this token when I made that comment (and still don't). In the image on that PR you can see where this background color is being used. To me it looks like a standard application background use case but I don't have the full context.

My preference after thinking about it some more would be for @WarriorOfNI to reach out to the designer that requested this color and see which category this color falls in:
1. it's a standard application background, in which case it should use the existing Nimble application background token (which maps to FFFFFF instead of E6E6E6)
2. it's a different type of background that already has a Nimble token (e.g. section background which is Black15 / F1F1F2), in which case it should use that token
3. it's a new type of background that could appear in other apps (e.g. sub-section background?), in which case we should rename the token in this PR to have a generic name
4. it's intentionally different from other NI apps, in which case it should indeed be a constant within the app and not added to Nimble. Ideally our apps wouldn't differ like this but perhaps the styling limitations/defaults of Microstrategy mean this is the best option.

Reply:
Some additional context discussed offline:

From Milan:
---
Hey  Ty Hudgins! Paul-Andrei Arhire mentioned that we should include some design tokens in nimble that the microstrategy styling is using.

What we want to avoid is having application specific tokens in nimble (the current proposal is calling the token ozoneLibraryBackgroundColor #E6E6E6), instead we would like tokens that align with the design system concepts so the tokens can be re-used across apps.

In the microstrategy app the background color is what is being referenced in this screenshot:
![image](https://user-images.githubusercontent.com/1588923/220959570-6680d19f-4bf5-4027-a378-63da848d4f43.png)

Is the new alternative background color well aligned with nimble? I added Brandon O'Keefe just in-case there are thoughts on different applications having different background colors and what the goals are.

Should the microstrategy app use nimble's applicationBackgroundColor tokens instead to be better aligned with other nimble applications? You can see the relationship between several of nimble's tokens as follows:
 
light:
![image](https://user-images.githubusercontent.com/1588923/220959798-474d671a-1223-4fb0-abff-cc3d4ad86389.png)

dark:
![image](https://user-images.githubusercontent.com/1588923/220959866-6a706a29-a370-4600-9475-462929754c74.png)

One concern I have with including the new proposed token #E6E6E6 is that if some apps use that instead of the expected applicationBackgroundColor they will run into styling issues (the current proposal says to use the same color in all theme modes):

![image](https://user-images.githubusercontent.com/1588923/220959962-d2310775-e881-4339-8f14-540c2a9069f5.png)

![image](https://user-images.githubusercontent.com/1588923/220960002-3d0f4673-7db7-4659-9a5d-976899f42622.png)

So concretely some questions are:

- Does the new token actually represent an alternate type of background color? Should microstrategy instead use nimble's built in applicationBackgroundColor?
- Does this usage in microstrategy actually behave more as a section background color? Should microstrategy instead use nimble's sectionBackgroundColor token?
- If this does really represent a new application background color, then what are the expected use-cases? What documentation needs to be given so applications know when and how to use it in relation to nimble's other styles? In addition, what should the token be configured for in the dark and the color themes (nimble's tokens are theme aware and as seen above using the same token value for all themes is incorrect)?

From Brandon:
---
Thanks Milan Raj, this is great feedback.  I've included some examples of color tokens as they would relate to Ty Hudgins in XD since the token structure is different and the names are not 1 to 1 with dev implementation.  All of this can be found on the Nimble_Components XD spec.  Also some of the examples here might be trailing a little behind what the header/Summary pane visual design will look like in final execution.

![image](https://user-images.githubusercontent.com/1588923/220960522-fe46ed6a-4569-46c7-b0f7-314a8469ba5b.png)

![image](https://user-images.githubusercontent.com/1588923/220960602-e5883171-6f1d-42d1-9e5e-fcdb6a234aad.png)

![image](https://user-images.githubusercontent.com/1588923/220960715-afa28152-389d-4a32-b65c-e4044dc08110.png)

![image](https://user-images.githubusercontent.com/1588923/220960794-31291f43-0c11-48b4-b002-69ed052f720a.png)

Reply:
Discussing with Ty, the resulting points are:
- Seems like there shouldn't be an issue with the microstrategy adopting the applicationBackgroundColor token instead to align with nimble
- The color `#E6E6E6` is used frequently in Brushs for apps such as in space after tables, between rows, etc. It also may be needed for the launch screen background in the future. However, we can add it to nimble once we have context of where it is used.

Reply:
@WarriorOfNI I'm going to close the current PR as it seems like we are not adding the new token at this time.